docs: Corellium sandbox (#1621)
Flank/flank: corellium/corellium-client/README.md

## About

Only for POC integration purposes. It's based on [corellium-api](https://github.com/corellium/corellium-api). It has

> only essential features implemented to be able to run simple iOS test run similar

Reviewer: The wording here is unclear?
Can you check it again.

Reviewer: Possibly:
```suggestion
only essential features implemented to be able to run a simple set of IOS tests that coincide 
```

```kotlin
val client = Corellium(
    api = String,
    username = String,
    password = String,
    tokenFallback = String,
    logging = LoggingLevel
)
```

#### logIn(): String (token)

Logs in with given credentials. Acquired `token` is persisted inside the client. It is also returned from the function
if you want to reuse it

```kotlin
val token: String = client.logIn()
```

#### getProjectIdList(): List<String>

Returns a list of project IDs associated with the given credentials

```kotlin
val ids: List<String> = client.getProjectIdList()
```

#### getAllProjects(): List<Project>

Returns a list of [Project](#data-class-project) associated with the given account

```kotlin
val projects: List<Project> = client.getAllProjects()
val testProjectId = projects.first { it.name == "test-project" }.id
```

#### getProjectInstancesList(projectId: String): List<Instance>

Returns a list of [Instance](#data-class-instance) created for the project with the given id

```kotlin
val projectId = "1111-2222-3333-aaa-bbbbbb-cccc"
val instances: List<Instance> = client.getProjectInstancesList(projectId)
```

#### createNewInstance(newInstance: Instance): String (instance id)

> Creates new [Instance](#data-class-instance), returns newly created instance id.

Reviewer: ```suggestion
Creates a new [Instance](#data-class-instance), returns newly created instance id.
```

NOTE: instance created with this method is not ready to use (state ==  `creating`).
Use [waitUntilInstanceIsReady](#waituntilinstanceisreadyinstanceid-string) to ensure VM has `on` state

> Following options are required (all `Instance` fields are described [here](#data-class-instance))

Reviewer: ```suggestion
The following options are required (all `Instance` fields are described [here](#data-class-instance))
```
